gitkraken-client/gitkraken-focus-view.md

```diff
@@ -0,0 +1,36 @@
+---
+
+title: Focus View
+description: Focus View presents an overview of your Pull Requests, Issues and WIPs.
+taxonomy:
+    category: gitkraken-client
+
+---
+
+The Focus View displays all of your Pull Requests, Issues, and Works In Progress (WIPs) relevant to you for a selected workspace. You have a search bar to filter the results and columns that represent different states of the PRs, Issues, and WIPs, and dropdown selectors for workspace-based filtering.
```

Comment:
> The Focus View displays all of your Pull Requests, Issues, and Works In Progress (WIPs) relevant to you for a selected ~workspace~ filters

This, along with many mentions of workspaces below, should probably be updated to reflect that there are more filters (issues, service and workspaces)